src/pushsource/_impl/model/erratum.py

```diff
@@ -354,6 +354,14 @@ class ErratumPushItem(PushItem):
     :type: list[str]
     """
 
+    appliance_image_list = attr.ib(
```

Comment:
I'm not convinced that this should be added as a public field on ErratumPushItem.

Do you have any plan to make use of it?  If so, can you share some details about it?  If no, please leave it out.  The info is anyway implied by the set of items you get back along with this item.

The problems I have with this field as currently defined (being a list of koji NVR):

(1) API shouldn't be unnecessarily tied to koji.

ErratumPushItem and VMIPushItem can already be generated without involving ET and koji at all (using a staging directory).  What would be put in this field in that case?  If it would be left blank - that means consumers have to be prepared to deal with a subtle difference between ErratumPushItem loaded from ET, and loaded from a staging dir, and they probably won't be aware of that.

(2) It's not consistent with or useful in the same way the other similar fields (pkglist for RPMs, and container_list for images) are useful.

Those other fields expose data which ends up going into customer portal and which makes sense to the end-user.  e.g. the names and checksums of RPMs makes sense, and the pull specs and manifest digests of container images make sense.  The NVRs of builds in koji don't make sense since that's an internal system the customers aren't aware of, don't have access to, and potentially could be replaced with something else.

TL;DR if you don't have an actual need for this, please don't add it; and if you *do* have a need for it then we need to discuss further what the format should be, because a list of koji NVRs is probably not good enough.

Reply:
I thought it was needed but it's not so I removed it.